### ISSUE TITLE
Let's start upgrading...

### ISSUE THREAD

**Issue by iamsitting** [attendanceproject/djattendance#1251]
- [ ] - Python 2 -> Python 3 (https://docs.djangoproject.com/en/1.11/topics/python3/)
  - [ ] Install packages (https://pypi.org/project/caniusepython3/)
  - [ ] Port code
- [ ] - Django 1.11 -> 2.0 (https://docs.djangoproject.com/en/2.0/howto/upgrade-version/)

**Comment by yipingbenjaminliu** [comment]
@billyeh , all you bro

**Comment by finben** [comment]
Yes, this is the main goal I think we need to have for service week.

**Comment by Plew** [comment]
This is being worked on in PR #1311.
